talk/owt/BUILD.gn

```diff
@@ -117,6 +117,7 @@ static_library("owt_sdk_base") {
     "sdk/base/vcmcapturer.h",
     "sdk/base/webrtcaudiorendererimpl.cc",
     "sdk/base/webrtcaudiorendererimpl.h",
+    "sdk/base/windowcapturer.cc",
```

Comment:
Why move it out of `is_win || is_linux`?

Reply:
The implementation of windowcapturer is OS independent. But since only Windows is using it. Will restore it to original code set.